[docs] CDCSDK: Update CDC documentation with missing sections (#21512)
yugabyte/yugabyte-db: docs/content/preview/reference/configuration/yb-master.md

Default: `14400` (4 hours)

##### --enable_truncate_cdcsdk_table

> By default, TRUNCATE commands on tables on which CDCSDK stream is active fail. Changing the value of this flag from `false` to `true` enables truncating the tables part of the CDCSDK stream.

Reviewer: ```suggestion
By default, TRUNCATE commands on tables with an active CDCSDK stream will fail. Change this flag to `true` to enable truncating tables.
```

Toggle automatic tablet splitting for tables in a CDCSDK stream, enhancing user control over replication processes.

##### --enable_truncate_cdcsdk_table

> By default, TRUNCATE commands on tables on which CDCSDK stream is active will fail. Changing the value of this flag from `false` to `true` will enable truncating the tables part of the CDCSDK stream.

Reviewer: ```suggestion
By default, TRUNCATE commands on tables with an active CDCSDK stream will fail. Change this flag to `true` to enable truncating tables. 
```
